src/Compilers/Core/CodeAnalysisTest/Analyzers/AnalyzerConfigTests.cs

```diff
@@ -2101,6 +2100,75 @@ public void GlobalConfigOptionsAreEmptyWhenNoGlobalConfig()
             Assert.Empty(globalOptions.TreeOptions);
         }
 
+        [Theory]
+        [InlineData("/path/to/file.cs", true)]
+        [InlineData("file.cs", false)]
+        [InlineData("../file.cs", false)]
+        [InlineData("**", false)]
+        [InlineData("*.cs", false)]
+        [InlineData("?abc.cs", false)]
+        [InlineData("/path/to/**", false)]
+        [InlineData("/path/[a]/to/*.cs", false)]
+        [InlineData("/path{", false)]
+        [InlineData("/path}", false)]
+        [InlineData("/path?", false)]
+        [InlineData("/path,", false)]
+        [InlineData("/path\"", true)]
+        [InlineData("/path\\", false)] //editorconfig sees a single escape character (special)
+        [InlineData("/path\\\\", true)] //editorconfig sees an escaped backslash
```

Comment:
>"/path\\\\" [](start = 20, length = 11)

The cases with backslashes are difficult to read with the extra escaping. Consider using verbatim string literals in those cases.